### ISSUE TITLE
GithubProfile.java:247-250: Implement validation using Kwalify library...

### ISSUE THREAD

**Issue by alex-palevsky** [yegor256/rultor#699]
Puzzle `570-b7bcc1c2` in `src/main/java/com/rultor/profiles/GithubProfile.java:247-250` has to be resolved: Implement validation using Kwalify library in separate class called ValidYaml, move this method to that class and move tests from GitHubProfileValidationTest to ValidYamlTest. Remember about removing PMD suppress below.

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by alex-palevsky** [comment]
@alexdag it's yours now, please proceed keeping in mind [our principles](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Feel free to ask any technical questions right here in the ticket

The cost of this task is **30 mins** (this is exactly how much will be paid, not less not more), when the task is [done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by alex-palevsky** [comment]
@alexdag the ticket is with you for 11 days already. If it's not finished in the next 48 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle; do not forget that you should remove the `@todo` puzzle from the source code; -30 added to your rating, at the moment it is: [-30](http://www.netbout.com/b/35099?open=rating)

**Comment by AlexDag** [comment]
@alex-palevsky , sorry for my surprised delay,I've some problem with my yahoo mail.So, please, give me 11 days and I'm starting with this task now. Thanks in advance.

**Comment by yegor256** [comment]
@alex-palevsky we need more time here

**Comment by alex-palevsky** [comment]
> @alex-palevsky we need more time here

@yegor256 of course, thanks for letting me know, take your time

**Comment by AlexDag** [comment]
@alex-palevsky  - we have #731 closed and does not any solutions for usage Kwalify.And Kwalify is a main target for this job.It's a technical question but- Does need to create a new ticket?Or change the specification of the job?

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear thanks, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear got it, someone else will be assigned soon

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #1032

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1032

@original-brownbear right, let's wait for #1032

**Comment by alex-palevsky** [comment]
@original-brownbear Closing, thanks all
